src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -129,28 +135,26 @@ public UserRole getUserRoleClassifications(String userRole) {
             final HttpEntity requestEntity = new HttpEntity<CaseType>(securityUtils.authorizationHeaders());
             final Map<String, String> queryParams = new HashMap<>();
             queryParams.put("roles", StringUtils.join(userRoles, ","));
-            return Arrays.asList(restTemplate.exchange(applicationParams.userRolesClassificationsURL(), HttpMethod.GET, requestEntity, UserRole[].class, queryParams).getBody());
+            return Arrays.asList(ofNullable(getClassificationForUserRoleList(requestEntity, queryParams)).orElseThrow(() -> NULL_RESPONSE_EX).getBody());
         } catch (Exception e) {
-            LOG.warn("Error while retrieving classification for user roles {} because of ", userRoles, e);
-            throw new ServiceException("Error while retrieving classification for user roles " + userRoles + " because of " + e.getMessage());
+            LOG.warn("Error while retrieving classification for user roles {} because of: ", userRoles, e);
+            throw new ServiceException("Error while retrieving classification for user roles " + userRoles + " because of: " + e.getMessage());
         }
     }
 
     @Override
     public CaseTypeDefinitionVersion getLatestVersion(String caseTypeId) {
         try {
             final HttpEntity requestEntity = new HttpEntity<CaseType>(securityUtils.authorizationHeaders());
-            CaseTypeDefinitionVersion version = restTemplate.exchange(applicationParams.caseTypeLatestVersionUrl
-                    (caseTypeId), HttpMethod.GET, requestEntity, CaseTypeDefinitionVersion.class).getBody();
+            CaseTypeDefinitionVersion version = ofNullable(getLatestVersion(caseTypeId, requestEntity)).orElseThrow(() -> NULL_RESPONSE_EX).getBody();
```

Comment:
I must admit I find it difficult to browse line longer than 120 characters on my screen

Reply:
I thought we agreed and later changed the line length to 160?